// This should throw because the void keyword was used as a function name
void void() {}